ampd/src/evm/verifier.rs

```diff
@@ -1,4 +1,5 @@
 use axelar_wasm_std::voting::Vote;
+use cast::usize;
```

Comment:
where is it used?

Reply:
https://github.com/axelarnetwork/axelar-amplifier/blob/34d4a541897c1a53a6f54c9b2f2be4487d619ebb/ampd/src/evm/verifier.rs#L75